src/frameworks/vite/index.ts

```diff
@@ -58,7 +53,9 @@ export async function discover(dir: string, plugin?: string, npmDependency?: str
   const anyConfigFileExists = configFilesExist.some((it) => it);
   if (!anyConfigFileExists && !findDependency("vite", { cwd: dir, depth, omitDev: false })) return;
   if (npmDependency && !additionalDep) return;
-  const { appType, publicDir: publicDirectory, plugins } = await getConfig(dir);
+  const config = await getConfig(dir);
+  if (!config) return;
+  const { appType, publicDir: publicDirectory, plugins } = config;
```

Comment:
I think that empty lines would make this easier to read
```suggestion

  const config = await getConfig(dir);
  if (!config) return;

  const { appType, publicDir: publicDirectory, plugins } = config;
```